File: tests/nodejs/cases/case_267_dead_sink_decoy_06.js
/**
 * Task case 267: dead_sink_decoy_06 deep stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case267DeadSinkDecoy06(input) {
  const taskName = 'dead_sink_decoy_06';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'extreme',
    stage: 'stress',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  if (1 === 2) { retainedAudit.push(payload); }
  // SAFE: dead branch only.
  return payload.checksum;
}

module.exports = { case267DeadSinkDecoy06 };

File: tests/go/cases/case_267_dead_sink_decoy_06.go
package escape_tests

var retainedCase267 = []map[string]string{}

func Case267DeadSinkDecoy06(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "dead_sink_decoy_06",
		"entity": "extreme",
		"stage": "stress",
		"input": raw,
		"checksum": "dead_sink_decoy_06:" + raw,
	}
	if 1 == 0 {
		retainedCase267 = append(retainedCase267, payload)
	}
	// SAFE: dead branch never executes sink.
	return payload["checksum"]
}
